Cloudinary/Classes/Core/CLDAnalytics.swift

```diff
@@ -35,15 +36,25 @@ import UIKit
                 techVersion = getiOSVersion()
             }
         }
+        if osVersion == nil {
```

Comment:
In case someone passes `osVersionString`, but `osVersion` is set( with a different value), you never use the parameter.
How you override the parameter?

Reply:
Fixed